src/libs/state.js

```diff
@@ -13,6 +13,8 @@ export const authStore = Utils.atom({
   fenceStatus: {}
 })
 
+export const cookieReadyStore = Utils.atom(false)
```

Comment:
For the sake of clarity/security/expression of intent, I think we should reset this to `false` on sign-out. This could be done when adding the call to clear the cookie, but I'd rather it be in this PR instead because this is where the global state is being added.